modules/operator-managed-postgres.adoc

```diff
@@ -1,6 +1,17 @@
 [[operator-managed-postgres]]
 = Using the managed PostgreSQL
 
+Requirements:
+
+If you are using an externally managed PostgreSQL database, you must manually enable `pg_trgm` extension for a successful deployment.
```

Comment:
Just a few small errors I've addressed below: 


If you are using an externally managed PostgreSQL database, you must manually enable `pg_trgm` extension for a successful deployment.
Enter the following command to enable the `pg_trgm` extension, replacing the name of the PostgreSQL pod with your pod: 
[source,terminal]
$ oc exec -it postgres-xxxxxxxxxx-xxxxx -n quay-enterprise -- /bin/bash -c 'echo "CREATE EXTENSION IF NOT EXISTS pg_trgm" | /opt/rh/rh-postgresql10/root/usr/bin/psql -d quay'

[NOTE]
====
The `-d`, or database name, must not be omitted. If omitted, the extension will be created on the default PostgreSQL database.
====